test/cmd/images.sh

```diff
@@ -64,7 +64,7 @@ oc describe istag/mysql:latest
 [ "$(oc describe istag/mysql:latest | grep "Image Created:")" ]
 [ "$(oc describe istag/mysql:latest | grep "Image Name:")" ]
 name=$(oc get istag/mysql:latest --template='{{ .image.metadata.name }}')
-imagename="isimage/mysql@${name:0:7}"
+imagename="isimage/mysql@${name}"
```

Comment:
> To verify that you can retrieve images by their short ID. Which we still want to do, but if there are two we should be able to choose the first. I'm not sure why there are 2 now though.

what is the "short id"?  what this code was grabbing was something like "mysql@sha:"

and there are now 2 because you have the mysql56 and mysql55 images, both of which start with that string.